pinot-common/src/main/java/org/apache/pinot/common/config/GrpcConfig.java

```diff
@@ -23,7 +23,15 @@
 import org.apache.pinot.common.utils.tls.TlsUtils;
 import org.apache.pinot.spi.env.PinotConfiguration;
 
-
+/// Configs used for the gRPC **query** service.
+///
+/// Remember that in Pinot we use different gPRC services for different purposes:
+/// - **query**: used by Pinot users for executing queries
```

Comment:
Looks like this class is also used for the internal broker -> server communication for the single-stage query engine when `pinot.broker.request.handler.type` is set to `grpc` (i.e., the `GrpcBrokerRequestHandler`)?